From: Virendra Pathak <virendra.pathak@broadcom.com>
Date: Mon, 13 Jun 2016 14:17:31 +0100
Subject: [PATCH] Accept vulcan as a cpu name for the AArch64 port of GAS.

       * config/tc-aarch64.c (aarch64_cpus): Add Broadcom Vulcan.
        * doc/c-aarch64.texi: Document that vulcan is a valid processor
	name.

Index: binutils-2.26/gas/config/tc-aarch64.c
===================================================================
--- binutils-2.26.orig/gas/config/tc-aarch64.c
+++ binutils-2.26/gas/config/tc-aarch64.c
@@ -7774,6 +7774,9 @@ static const struct aarch64_cpu_option_t
   {"thunderx", AARCH64_FEATURE (AARCH64_ARCH_V8,
 				AARCH64_FEATURE_CRC | AARCH64_FEATURE_CRYPTO),
    "Cavium ThunderX"},
+  {"vulcan", AARCH64_FEATURE (AARCH64_ARCH_V8,
+			      AARCH64_FEATURE_CRC | AARCH64_FEATURE_CRYPTO),
+  "Broadcom Vulcan"},
   /* The 'xgene-1' name is an older name for 'xgene1', which was used
      in earlier releases and is superseded by 'xgene1' in all
      tools.  */
Index: binutils-2.26/gas/doc/c-aarch64.texi
===================================================================
--- binutils-2.26.orig/gas/doc/c-aarch64.texi
+++ binutils-2.26/gas/doc/c-aarch64.texi
@@ -62,6 +62,7 @@ on the target processor.  The following
 @code{exynos-m1},
 @code{qdf24xx},
 @code{thunderx},
+@code{vulcan},
 @code{xgene1}
 and
 @code{xgene2}.
